snforge_std/src/cheatcodes.cairo

```diff
@@ -109,3 +109,22 @@ fn stop_mock_call(contract_address: ContractAddress, function_name: felt252) {
     let contract_address_felt: felt252 = contract_address.into();
     cheatcode::<'stop_mock_call'>(array![contract_address_felt, function_name].span());
 }
+
+
+fn store(target: ContractAddress, storage_address: felt252, serialized_value: Span<felt252>) {
+    let mut inputs = array![target.into(), storage_address];
+    inputs.append_span(serialized_value);
+
+    cheatcode::<'store'>(inputs.span());
+}
+
+fn load(target: ContractAddress, storage_address: felt252, size: felt252) -> Span<felt252> {
```

Comment:
It would be useful if the user could specify the type returned by the `load` function

Reply:
I wanted to do it this way but that would require the types to be constrained by some traits which allows us to:
- Deserialize (serde)
- Calculate onchain type size (in order to load it properly). <- This one does not exist, there isn't a standard one that we can use

Reply:
How about leaving the size argument here and just deserialize extracted data with serde?